docs/guides/social-signin/get-tokens.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: get-tokens
+title: Get Social Sign-in Provider Tokens
+---
+
+You can get the OIDC / OAuth 2.0 access, refresh, and ID tokens issued for the Identity by social sign-in providers. 
```

Comment:
Same question - did you make modifications (i.e. do i need to review)?

Reply:
Have a quick look. It's moved from one place to another but I added some information (like the CLI bit)